code/game/jobs/job_exp.dm

```diff
@@ -0,0 +1,192 @@
+// Admin Verbs
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><body>Playtime:<BR><UL>"
+	for(var/client/C in clients)
+		msg += "<LI> - [key_name_admin(C)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_living() + "</a></LI>"
+	msg += "</UL></BODY></HTML>"
+	src << browse(msg, "window=Player_playtime_check")
+
+
+/datum/admins/proc/cmd_show_exp_panel(var/client/C)
+	if(!C)
+		to_chat(usr, "ERROR: Mob not found.")
+		return
+	if(!check_rights(R_ADMIN))
+		return
+	var/body = "<html><head><title>Playtime for [C.key]</title></head><BODY><BR>Playtime:"
+	body += C.get_exp_report()
+	body += "</BODY></HTML>"
+	usr << browse(body, "window=playerplaytime[C.ckey];size=550x615")
+
+
+// Procs
+
+
+/datum/job/proc/available_in_playtime(client/C)
```

Comment:
This proc returns how much playtime until the job is available. 0 means "0 minutes until it's available".